PROTOCOL.md

```diff
@@ -1876,7 +1881,11 @@ The request body should be a JSON string containing the following two optional f
 - **limitHint** (type: Int32, optional): an optional limit number. It’s a hint from the client to tell the server how many rows in the table the client plans to read. The server can use this hint to return only some files by using the file stats. For example, when running `SELECT * FROM table LIMIT 1000`, the client can set `limitHint` to `1000`.
   - Applying `limitHint` is **BEST EFFORT**. The server may return files containing more rows than the client requests.
 
-- **version** (type: Int64, optional): an optional version number. If set, will return files as of the specified version of the table. This is only supported on tables with change data feed (cdf) enabled. 
+- **version** (type: Int64, optional): an optional version number. If set, will return files as of the specified version of the table. This is only supported on tables with history sharing enabled.
+
+- **timestamp** (type: String, optional): an optional timestamp string. If set, will return files as of the table version corresponding to the specified timestamp. This is only supported on tables with history sharing enabled.
```

Comment:
Consider adding details about the time zone for all the timestamp related params. Are the timestamps expected to be UTC?

Reply:
added a `Timestamp Format` section.